src/Features/Demo/DemoGhostPlayer.cpp

```diff
@@ -224,7 +224,7 @@ CON_COMMAND_AUTOCOMPLETEFILE(ghost_set_demo, "ghost_set_demo <demo> [ID]. Ghost
     sf::Uint32 ID = args.ArgC() > 2 ? std::atoi(args[2]) : 0;
     demoGhostPlayer.DeleteGhostsByID(ID);
     if (demoGhostPlayer.SetupGhostFromDemo(engine->GetGameDirectory() + std::string("/") + args[1], ID, false)) {
-        console->Print("Ghost sucessfully created ! Final time of the ghost : %s\n", SpeedrunTimer::Format(demoGhostPlayer.GetGhostByID(ID)->GetTotalTime()).c_str());
+        console->Print("Ghost successfully created ! Final time of the ghost : %s\n", SpeedrunTimer::Format(demoGhostPlayer.GetGhostByID(ID)->GetTotalTime()).c_str());
```

Comment:
if you're up for it, it'd be cool if you could also fix the grammar in strings like this - there shouldn't be whitespace before periods / exclamation marks / question marks / colons
i.e. this string should become `"Ghost successfully created! Final time of the ghost: %s\n"`

Reply:
I'll take a look at this